README.md

```diff
@@ -1,6 +1,7 @@
 Topic:- Bike Info
 The overview of the website:- On this website, clicking on a company logo provides detailed information about each bike, including the manufacturer's country, top speed, weight, mileage, engine capacity, price, and more.
 
+fg
```

Comment:
Remove the added line 'fg' as it doesn't provide any meaningful information. [important]